content/en/hugo-pipes/introduction.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Hugo Pipes Introduction
+description: Hugo Pipes is Hugo's asset processing set of functions.
+date: 2018-07-14
+publishdate: 2018-07-14
+lastmod: 2018-07-14
+categories: [asset management]
+keywords: []
+menu:
+  docs:
+    parent: "pipes"
+    weight: 20
+weight: 01
+sections_weight: 01
+draft: false
+aliases: [/assets/]
+--- 
+
+### Asset directory
+
+Asset files must be stored in the asset directory which defaults to `/assets` but can be configured via the configuration file's `assetDir` key.
+
+### From file to resource
+
+In order to process an asset with Hugo Pipes it must be retrieved as a resource using `resources.Get` which takes one argument, the filepath of the file relative to the asset directory.
+
+```go-html-template
+{{ $style := resoursec.Get "sass/main.scss" }}
+```
+
+### Asset publishing
+
+Assets will only be published to the project's publish directory if their `.Permalink` or `.RelPermalink` is outputted in one of the project's template file.
```

Comment:
I'm always torn between taking into account everyones own configuration (not everyone uses /public) and simple sentences... But the I agree here the simple sentence wins.

Reply:
My main point was avoiding the "project", because then you would have to add theme to the mix. But I find that everyone that has touched Hugo knows about  /public, even those who have reconfigured it.

Reply:
> It would maybe be good with a little info about the /resources directory, but that can wait.

I will leave you or someone else phrase that in, I've never investigated this too much.